jerry-debugger/jerry-client-ws.py

```diff
@@ -273,7 +274,19 @@ def do_step(self, args):
 
     def do_next(self, args):
         """ Next breakpoint in the same context """
-        self.exec_command(args, JERRY_DEBUGGER_NEXT)
+        if args == "":
+            args = 0
```

Comment:
The preferred way to check that a string is empty (if you know that your variable is a string) is this: `if not args:`

If your variable could also be some other type then you should use this form `if args == "":`, but as I saw in the following code, it always waiting for string and then you cast that into an integer.